package.json

```diff
@@ -59,14 +59,14 @@
   "bugs": {
     "url": "https://github.com/softonic/axios-retry/issues"
   },
-  "types": "./index.d.ts",
-  "main": "index.js",
-  "module": "lib/esm/index.js",
+  "types": "dist/index.d.ts",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
   "exports": {
     ".": {
-      "types": "./index.d.ts",
-      "import": "./lib/esm/index.js",
-      "require": "./index.js"
+      "types": "./dist/index.d.ts",
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
```

Comment:
@mindhells 

This setup would result in the following error.

```console
$ npx tsc
srv/app.ts:5:1 - error TS2349: This expression is not callable.
  Type 'typeof import("/home/study/workspace/ts-node-oidc/node_modules/axios-retry/dist/index")' has no call signatures.

5 axiosRetry(axios);
  ~~~~~~~~~~
```

A draft amendment is prepared below. We hope you will review it.

https://github.com/softonic/axios-retry/pull/255